docs/source/Analysis/queries.md

```diff
@@ -202,26 +172,22 @@ Running this algorithm returns the following data:
 32670,Celebrimbor,2
 32670,Grimbeorn,2
 32670,Lobelia,2
+...
 ```
 
-This data tells us that at a given time, person X and Gandalf are N number of hops away.
-For example, at time 32670, Samwise was at minimum 1 hop away from Gandalf, whereas Lobelia was 2 hops away.
+This data tells us that at a given time, person X and Gandalf are N number of hops away. In this instance, at time 32670, Samwise was at minimum 1 hop away from Gandalf, whereas Odo was 2 hops away.
 
 ## Using Raphtory as a client
 
-Finally, if you have a graph deployed somewhere else and want to submit new queries to it,
-you can do this via the `deployedGraph(customConfig)` method in `Raphtory` object,
-providing the appropriate configuration to locate it.
-If the graph is deployed in the same machine using the default configuration,
-you can even omit the configuration parameter:
+Finally, if you have a graph deployed somewhere else and want to submit new queries to it you can do this via the `deployedGraph(customConfig)` method in the `Raphtory` object. The `customConfig` here is to provide the appropriate configuration to locate the graph (i.e. the akka/pulsar address). If the graph is deployed in the same machine using the default Raphtory configuration you can omit this configuration parameter:
 
```

Comment:
> If the graph is deployed in the same machine using the default Raphtory configuration you can omit this configuration parameter

This is not true right now

Reply:
Yeah this needs to be moved into the deployment section along with the update to streaming/batchload